### PR TITLE
Update README to fix broken TF link

### PR DESCRIPTION
**Pull request by indie**
+ Group edits from PR 2994 content and illustrations added to avert conflicts, 
+ Make new subsections for extras like HE Transformer and Distributed Training options
+ Refactor ToC, ensuring all links work and no dead links.

### CODE REVIEW

File: doc/sphinx/source/frameworks/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+You can install TensorFlow\* and nGraph to a virtual environment; otherwise, the code 
+will install to a system location.
+
+.. code-block:: console
+   
+   pip install tensorflow
+   pip install ngraph-tensorflow-bridge
+
+.. note:: You may need to check the latest versions of ```tensorflow``` and the bridge
```
Comment (diyessi) [inline doc/sphinx/source/frameworks/getting_started.rst]: `check` -> `use`

Comment (indie) [reply doc/sphinx/source/frameworks/getting_started.rst]: :+1:

File: doc/sphinx/source/project/extras/homomorphic_encryption.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.. project/extras/homomorphic_encryption.rst:
+
+Homomorphic Encryption (HE)
+===========================
+
+* **Encryption with Intel® HE transformer for nGraph™** 
+
+  * The `Intel HE_transformer`_ enables deep encryption with nGraph Backends.
```
Comment (fboemer) [inline doc/sphinx/source/project/extras/homomorphic_encryption.rst]: Could we change this to ```The `Intel HE_transformer`_ is an experimental nGraph backend which enables deep learning on encrypted data using homomorphic encryption.```

Comment (indie) [reply doc/sphinx/source/project/extras/homomorphic_encryption.rst]: :+1:

**Comment by indie** [comment]
These changes should be added ASAP to help us get ready for the ngraph.ai launch which Bethany Dalby and I are working on today.

**Pull request merged**